services/navigation/builtin/builtin.go

```diff
@@ -495,42 +509,86 @@ func (svc *builtIn) Close(ctx context.Context) error {
 	return svc.store.Close(ctx)
 }
 
+func (svc *builtIn) moveOnGlobeSync(ctx context.Context, wp navigation.Waypoint, extra map[string]interface{}) error {
+	req := motion.MoveOnGlobeReq{
+		ComponentName:      svc.base.Name(),
+		Destination:        wp.ToPoint(),
+		Heading:            math.NaN(),
+		MovementSensorName: svc.movementSensor.Name(),
+		Obstacles:          svc.obstacles,
+		MotionCfg:          svc.motionCfg,
+		Extra:              extra,
+	}
+	var stopWG sync.WaitGroup
+	defer stopWG.Wait()
+	cancelCtx, cancelFn := context.WithCancel(ctx)
+	defer cancelFn()
+	rawExecutionID, err := svc.motionService.MoveOnGlobeNew(cancelCtx, req)
+	if err != nil {
+		return err
+	}
+
+	executionID, err := uuid.Parse(rawExecutionID)
```

Comment:
Why is this necessary?

Reply:
b/c `MoveOnGlobeNew`returns a `string,error`, where the string is a stringified executionID, which is a uuid, but `motion.PlanHistory` takes the a non-stringified executionID. Hence there is need to parse the rawExeuctionID (string) into a uuid.

Reply:
Can't we just change MoveOnGlobeNew to return the executionID directly?  The client server code can handle the conversion to and from the string that it must exist as in proto

Reply:
I'll get started on that now

Reply:
done